src/cms/preview-templates/IndexPagePreview.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import { IndexPageTemplate } from "../../templates/index-page";
 
-const IndexPagePreview = ({ entry, getAsset }) => {
+const IndexPagePreview = ({ entry }) => {
   const data = entry.getIn(["data"]).toJS();
```

Comment:
You make me realise about a bug. https://github.com/debtcollective/student-debt-campaign-netlify/issues/17 I am removing all this code because is not being used.